tests/tcp.rs

```diff
@@ -205,12 +205,8 @@ fn read() {
         for event in &events {
             assert_eq!(event.token(), Token(1));
             let mut buf = [0; 1024];
-            loop {
-                if let Ok(amt) = data.socket.read(&mut buf) {
-                    data.amt += amt;
-                } else {
-                    break;
-                }
+            while let Ok(amt) = data.socket.read(&mut buf) {
+                data.amt += amt;
```

Comment:
This change seems unrelated to the windows-sys change.

Reply:
See my answer here: https://github.com/tokio-rs/mio/pull/1857#discussion_r1925522574